HotelManagement/src/main/java/com/deltasource/hotelmanagement/core/HotelServiceApplication.java

```diff
@@ -1,37 +1,43 @@
-import core.*;
-import core.Commodities.Bed;
-import core.Commodities.Shower;
-import core.Commodities.Toilet;
+package com.deltasource.hotelmanagement.core;
+
+import com.deltasource.hotelmanagement.core.commodities.Bed;
```

Comment:
You can import all of them with com.deltasource.hotelmanagement.core.commodities.*;